packages/manager/modules/telecom-dashboard/src/ftth-eligibility/telecom-dashboard-ftth-eligibility.controller.js

```diff
@@ -0,0 +1,94 @@
+import {
+  ELIGIBILITY,
+  ACCESS_TYPE,
+  CLOSURE_DETAIL_URL,
+} from './telecom-dashboard-ftth-eligibility.constant';
+
+export default class FtthEligibilityCtrl {
+  /* @ngInject */
+  constructor(coreURLBuilder, FtthEligibilityService, TucToastError) {
+    this.coreURLBuilder = coreURLBuilder;
+    this.FtthEligibilityService = FtthEligibilityService;
+    this.TucToastError = TucToastError;
+    this.ELIGIBILITY = ELIGIBILITY;
+    this.ACCESS_TYPE = ACCESS_TYPE;
+  }
+
+  $onInit() {
+    this.links = {
+      accessList: this.coreURLBuilder.buildURL('telecom', '#/xdsl-access-list'),
+      copperClosureMore: CLOSURE_DETAIL_URL,
+    };
+    this.amountServicesDisplayed = 5;
+    this.servicesLength = 0;
+
+    this.getAllServices();
+  }
+
+  getAllServices() {
+    this.services = [];
```

Comment:
Why initializing this.services array here if it used in createServiceList then?

Wouldn't it be more safe to initialize it in createServiceList ?